### PR TITLE
Changed unfolding of arithmetics from right to left

### PR DESCRIPTION
**Pull request by triskaj**

### CODE REVIEW

File: t/lang/compiler/compile-expression.lisp [hunk 1]
```diff
@@ -180,7 +180,7 @@
 
 (let ((var-env (empty-variable-environment))
       (func-env (empty-function-environment)))
-  (is (compile-arithmetic '(+ 1 1 1) var-env func-env) "(1 + (1 + 1))"
+  (is (compile-arithmetic '(+ 1 1 1) var-env func-env) "((1 + 1) + 1)"
       "basic case 1"))
```
Comment (guicho271828) [inline t/lang/compiler/compile-expression.lisp]: wouldnt it better to add the minus version here?

Comment (triskaj) [reply t/lang/compiler/compile-expression.lisp]: I had the same feeling. I think you're right.

Comment (takagi) [reply t/lang/compiler/compile-expression.lisp]: Would you add the case?

Comment (takagi) [reply t/lang/compiler/compile-expression.lisp]: Would you add the case?

**Comment by takagi** [comment]
Thanks a lot!

**Comment by triskaj** [comment]
You're welcome! Sorry that I didn't add the `-` test case, I wanted to do it today :-D.

______________________________________________________________

**Pull request merged**